regulations/static/regulations/js/source/views/comment/comment-review-view.js

```diff
@@ -27,19 +29,41 @@ var CommentReviewView = Backbone.View.extend({
 
     this.previewLoading = false;
 
+    this.listenTo(CommentEvents, 'read:proposal', this.handleRead);
+
     this.render();
   },
 
   findElms: function() {
     this.$status = this.$el.find('.status');
   },
 
+  handleRead: function() {
+    var section = this.docId + '-preamble-' + this.docId + '-I';
+    var options = {id: section, section: section, mode: 'read'};
+
+    $('#content-body').removeClass('comment-review-wrapper').addClass('preamble-wrapper');
```

Comment:
I didn't double check that `preamble-wrapper` was being used, but `comment-review-wrapper` does need to be on `content-body` to resize the width of the content area.